python/core/auto_generated/qgsvectordataprovider.sip.in

```diff
@@ -53,6 +53,10 @@ of feature and attribute information from a spatial datasource.
       CancelSupport,
       CreateRenderer,
       CreateLabeling,
+      CanDisableSelectAtId,
+      CanUseEstimatedMetadata,
+      CanDisbaleCheckPrimaryKeyUnicity,
```

Comment:
```suggestion
      CanDisableCheckPrimaryKeyUnicity,
```

Reply:
Thanks for the review. I have fixed the typo and add since it the flags description.